### PR TITLE
Update InactiveAccounts.yaml

### PR DESCRIPTION
**Pull request by shainw**
Required items, please complete
   
   Change(s):
   - Added filtering with UEBA to provide more concrete indication of something interesting along with provide UEBA context

   Reason for Change(s):
   - Results were highly noisy

   Version Updated:
   - Hunting query so not required

   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   - Will check validations and fix any issues.

### CODE REVIEW

File: Hunting Queries/SigninLogs/InactiveAccounts.yaml [hunk 1]
```diff
@@ -19,30 +19,57 @@ query: |
...
+  ) on $left.UserId == $right.NewUserId
   | extend timestamp = StartTime, AccountCustomEntity = UserPrincipalName
 
 entityMappings:
   - entityType: Account
     fieldMappings:
       - identifier: FullName
-        columnName: AccountCustomEntity
\ No newline at end of file
+        columnName: AccountCustomEntity
```
Comment (aprakash13) [inline Hunting Queries/SigninLogs/InactiveAccounts.yaml:75]: Wondering if we should also have an IP Entity in here that might be surfaced through BehaviorAnalytics table.

File: Hunting Queries/SigninLogs/InactiveAccounts.yaml [hunk 2]
```diff
@@ -19,30 +19,57 @@ query: |
...
-     | summarize by UserPrincipalName, Identity
-  ) on UserPrincipalName
+     | summarize by UserId
+  ) on UserId;
+  // need to help BehaviorAnalytics query to limit only to Signins we are interested in
+  let onlyInactive = SigninsSummary | summarize make_set(UserPrincipalName);
+  let SigninsWithUEBA =
+  BehaviorAnalytics
+  | where TimeGenerated between(starttime..endtime)
+  | where ActionType in ('Sign-in','InteractiveLogon')
```
[the remark this answers is not in the document]
Comment (shainw) [reply Hunting Queries/SigninLogs/InactiveAccounts.yaml:39]: This is a label within BehaviorAnalytics, we will potentially miss matches if we don't include it.

File: Hunting Queries/SigninLogs/InactiveAccounts.yaml (hunk 1 above)
Comment (shainw) [reply Hunting Queries/SigninLogs/InactiveAccounts.yaml:75]: Added, not used yet with hunting queries in the UX but definitely good to have it in place

File: Hunting Queries/SigninLogs/InactiveAccounts.yaml [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
-  with stale/inactive accounts that ought to have been deleted but were not - and have
-  subseuqently been compromised.
-  Results for user accounts created in the last 7 days are filtered out'
+  'Query for accounts seen signing in for the first time - these could be associated with stale/inactive accounts that ought to have been deleted 
+  but were not and may have been subseuqently been compromised.
+  UEBA is used to filter out based on ActivityInsights where we see certain Firt Time User events identified as true.
+  Results for user accounts created in the last 7 days are filtered out.'
 requiredDataConnectors:
   - connectorId: AzureActiveDirectory
     dataTypes:
```
[the remark this answers is not in the document]
Comment (shainw) [reply Hunting Queries/SigninLogs/InactiveAccounts.yaml:10]: Yes, and BehaviorAnalytics, good catch.

File: Hunting Queries/SigninLogs/InactiveAccounts.yaml [hunk 4]
```diff
@@ -1,10 +1,10 @@
 id: 847c2652-547d-4d5f-9b71-d2f8d81eac62
 name: Inactive or new account signins
 description: |
-  'Query for accounts seen signing in for the first time - these could be associated
-  with stale/inactive accounts that ought to have been deleted but were not - and have
-  subseuqently been compromised.
-  Results for user accounts created in the last 7 days are filtered out'
+  'Query for accounts seen signing in for the first time - these could be associated with stale/inactive accounts that ought to have been deleted 
+  but were not and may have been subseuqently been compromised.
+  UEBA is used to filter out based on ActivityInsights where we see certain Firt Time User events identified as true.
```
[the remark this answers is not in the document]
Comment (shainw) [reply Hunting Queries/SigninLogs/InactiveAccounts.yaml:6]: Done

**Pull request merged**